addons/html_builder/static/src/core/building_blocks/builder_colorpicker.js

```diff
@@ -12,6 +12,7 @@ export class BuilderColorPicker extends Component {
     static template = "html_builder.BuilderColorPicker";
     static props = {
         ...basicContainerBuilderComponentProps,
+        noTransparency: { type: Boolean, optional: true },
```

Comment:
hasTransparency 
defaultProps: { hasTransparency: true} 

negation props is not cool